localstack/services/awslambda/lambda_executors.py

```diff
@@ -1277,6 +1286,7 @@ def _execute_in_custom_runtime(
 
         process = run(cmd, stderr=subprocess.PIPE, outfile=subprocess.PIPE, **kwargs)
         result, log_output = process.communicate()
+
         try:
             result = to_str(result).strip()
         except Exception:
```

Comment:
ditto, the TODO seems to be pointing out that this code is not needed.

Reply:
Which TODO?

Reply:
https://github.com/localstack/localstack/blob/610924cb71beb49551a04e8d1dad747eb99c840f/localstack/services/awslambda/lambda_executors.py#L1289

Reply:
Since we are basically executing code here without the lambci wrapping, it might actually be the case for the local executor that logs are tangled up like that. I am opting for keeping it here to play it safe.

Reply:
Sounds good, the cleanup seems to be on progress anyway :)